### PR TITLE
adding auto encoder stat arb to fsi-samples repo

### PR DESCRIPTION
**Pull request by MarkJosephBennett**
Train a TensorFlow-based deep autoencoder on log returns for the Dow 30 securities in the initial minutes for 2018 for use subsequently to help make long/short trade entry decisions the rest of the year. Take advantage of temporary mispricings.

### CODE REVIEW

**Comment by GPUtester** [comment]
Can one of the admins verify this patch?

File: autoencoderstatarb/docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+RUN pip install -r /workspace/requirements.txt
+
+WORKDIR /rapids/notebooks
+
+ADD ./LICENSE /.
+ADD ./README.md ./
+ADD ./CONTRIBUTING.md ./
+ADD ./*.ipynb ./
+ADD ./*.png ./
+
```
Comment (yidong72) [inline autoencoderstatarb/docker/Dockerfile:37]:
```
ADD ./*.png ./
```
doesn't work
I have to do something like 
```--- a/autoencoderstatarb/docker/Dockerfile
+++ b/autoencoderstatarb/docker/Dockerfile
@@ -33,7 +33,7 @@ ADD ./LICENSE /.
 ADD ./README.md ./
 ADD ./CONTRIBUTING.md ./
 ADD ./*.ipynb ./
-ADD ./*.png ./
+ADD images ./images

 EXPOSE 8888
 EXPOSE 8889
```

Comment (yidong72) [reply autoencoderstatarb/docker/Dockerfile:37]: in the cuDAESA.ipynb file, the image path need to be consistent. either all under images directory or all in the '.' directory
see both
```
<br>
<img src=nvidialogo.png/>
<br>
```
```
<br>
<img src="images/image001.png" width="1200"/>
<br>
```

**Review by yidong72 — COMMENTED** [review]
Need to fix the dockerfile and image path in the notebooks.

**Pull request merged**